src/toast/atm.py

```diff
@@ -308,13 +312,16 @@ def simulate(self, use_cache=False, smooth=False):
             comm_node_rank=self._node_rank_comm,
         )
 
+        mem_shared = self._nelem * 8 / 2**20
         log.debug_rank(
-            "Allocated shared realization buffer in", comm=self._comm, timer=timer
+            f"Allocated {mem_shared:.1f}MB for shared realization buffer in",
+            comm=self._comm,
+            timer=timer,
         )
 
         ind_start = 0
         ind_stop = 0
-        slice = 0
+        slice_ = 0
```

Comment:
[PEP 8](https://peps.python.org/pep-0008/#method-names-and-instance-variables) (and most python code) uses a leading underscore (not trailing) to indicate private variables.

Reply:
Trailing underscores are used when reserved words are used as variables (also from PEP-8): 
![image](https://github.com/hpc4cmb/toast/assets/596250/31379d22-e189-47b9-8aca-4cb1351f455f)

Reply:
Got it, thanks!